govcd/vapp_test.go

```diff
@@ -541,7 +541,20 @@ func (vcd *TestVCD) Test_AddNewVMMultiNIC(check *C) {
 		fmt.Println("Skipping adding another vdc network as network2 was not specified")
 	}
 
-	task, err := vapp.AddNewVM(check.TestName(), vapptemplate, desiredNetConfig, true)
+	var task Task
+	var sp types.Reference
+	var customSP = false
+
+	if vcd.config.VCD.StorageProfile.SP1 != "" {
+		sp, _ = vcd.vdc.FindStorageProfileReference(vcd.config.VCD.StorageProfile.SP1)
+	}
+
+	if sp.HREF != "" {
```

Comment:
Maybe worth printing out the message when `testVerbose` is enabled.

Reply:
Thanks. Done